yarn.lock

```diff
@@ -61,6 +61,10 @@ acorn@^5.0.0, acorn@^5.3.0, acorn@^5.5.0:
   version "5.5.3"
   resolved "https://registry.yarnpkg.com/acorn/-/acorn-5.5.3.tgz#f473dd47e0277a08e28e9bec5aeeb04751f0b8c9"
 
+acorn@~5.4.0:
```

Comment:
I don't usually use `yarn` so I have no clue if I updated this properly (just letting `yarn add` do it's thing)

Reply:
Everything good.